azureadgraph.json

```diff
@@ -10,12 +10,12 @@
     "product_version_regex": ".*",
     "publisher": "Splunk",
     "license": "Copyright (c) 2019-2022 Splunk Inc.",
-    "app_version": "2.3.0",
+    "app_version": "2.4.0",
```

Comment:
Seeing that there is no feature change here, shouldn't the version be updated to `2.3.1`?

Reply:
Yes, As only document has been updated. So, Revert back the version as per version guidelines.